syne_tune/optimizer/schedulers/searchers/bayesopt/gpautograd/learncurve/freeze_thaw.py

```diff
@@ -197,10 +197,10 @@ def resource_kernel_likelihood_precomputations(targets: List[np.ndarray]) -> Dic
     assert num_configs[-1] > 0, num_configs
     total_size = sum(num_configs)
     assert total_size == sum(ydims)
-    # Attention: When comparing this to `issm.issm_likelihood_precomputations`,
-    # `targets` maps to `yflat` in the same ordering (the index is still r),
-    # whereas there the index j of `deltay` runs in the opposite direction of
-    # the index r of `targets`
+    # Attention: When comparing this to ``issm.issm_likelihood_precomputations``,
+    # ``targets`` maps to ``yflat`` in the same ordering (the index is still r),
+    # whereas there the index j of ``deltay`` runs in the opposite direction of
+    # the index r of ``targets``
```

Comment:
Do we want the double backticks in comments like this, or just in the docstrings?

Reply:
We should do the same in all comments. Of course, in non doc-string comments, you can do what you like.